# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xf86-input-fpit/0003-fpit-minX-maxX-get-wrongly-initialized.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From d5f2ccda167ee783e6fd8eaff31a1e16013a1585 Mon Sep 17 00:00:00 2001
From: philip <feucht@uni-koblenz.de>
Date: Wed, 19 May 2010 12:37:05 +0200
Subject: [PATCH 3/3] fpit: minX/ maxX get wrongly initialized

maxX/ minX get values from maxY/ minY
Problem introduced in commit 7d203627e7e3e7a6f8d0e847ed650b0b89760c09

Reviewed-by: Gaetan Nadon <memsize@videotron.ca>
Signed-off-by: philip <feucht@uni-koblenz.de>
---
 src/xf86Fpit.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/xf86Fpit.c b/src/xf86Fpit.c
index ce7540b..4c09c96 100644
--- a/src/xf86Fpit.c
+++ b/src/xf86Fpit.c
@@ -232,7 +232,7 @@ static void xf86FpitSetUpAxes(DeviceIntPtr dev, FpitPrivatePtr priv)
 #if GET_ABI_MAJOR(ABI_XINPUT_VERSION) >= 7
 				       axis_labels[0],
 #endif
-				       priv->fpitMinY, priv->fpitMaxY, 9500, 0 /* min_res */ ,
+				       priv->fpitMinX, priv->fpitMaxX, 9500, 0 /* min_res */ ,
 				       9500 /* max_res */ );
 		InitValuatorAxisStruct(dev, 1,
 #if GET_ABI_MAJOR(ABI_XINPUT_VERSION) >= 7
-- 
1.6.5.3

